modules/localgov_services_navigation/tests/src/FunctionalJavascript/EntityReferenceServicesAutocompleteTest.php

```diff
@@ -215,7 +215,7 @@ public function testEntityReferenceSublandingLabelWidget() {
     ]);
 
     $this->drupalGet('node/' . $node->id() . '/edit');
-    $this->assertSession()->fieldValueEquals('edit-localgov-services-parent-0-target-id', 'Landing Page 1 » Page Sub 1 (2)');
+    $this->assertSession()->fieldValueEquals('edit-localgov-services-parent-0-target-id', 'Page Sub 1 (2)');
```

Comment:
So this is actually a bug I think, so the 10 test should fail not the 9 test. It should show the hierarchy.

Reply:
https://github.com/localgovdrupal/localgov_services/pull/212/commits/19080ff79a6e388c10585bf488c0fd98af85ff6b hopefully